### PR TITLE
Add rails task to create workflow token and runs

### PR DESCRIPTION
**Pull request by saraycp**
With this task, we can easily fill in our development database with a workflow token and some workflow runs.
Data that helps us to try out things during development and in review-app.

### CODE REVIEW

**Review by danidoni — APPROVED** [review]

File: src/api/lib/tasks/dev.rake [hunk 1]
```diff
@@ -249,6 +249,65 @@ namespace :dev do
...
+      source_project = find_or_create_project(admin.home_project_name, admin)
+      target_project = find_or_create_project('home:Admin:CI:iggy:hello_world:PR-1', admin)
+
+      source_package = source_project.packages.find_by(name: 'hello_world') || create(:package_with_file, name: 'hello_world', project: source_project, file_name: 'README.md', file_content: '# Lorem ipsum')
+      target_package = target_project.packages.find_by(name: 'hello_world') || create(:package_with_file, name: 'hello_world', project: target_project, file_name: 'README.md', file_content: '# Lorem ipsum')
+
+      token = Token::Workflow.find_by(name: 'Testing token') || create(:workflow_token, user: admin, name: 'Testing token')
+
+      ['running', 'success', 'fail'].each do |status|
+        workflow_run = create(:workflow_run, token: token, status: status)
```
Comment (dmarcoux) [inline src/api/lib/tasks/dev.rake:280]: Would it be possible to also have data in `response_body` and `response_url`?

File: src/api/lib/tasks/dev.rake [hunk 2]
```diff
@@ -249,6 +249,65 @@ namespace :dev do
...
+    task data: :environment do
+      unless Rails.env.development?
+        puts "You are running this rake task in #{Rails.env} environment."
+        puts 'Please only run this task with RAILS_ENV=development'
+        puts 'otherwise it will destroy your database data.'
+        return
+      end
+
+      require 'factory_bot'
+      include FactoryBot::Syntax::Methods
```
Comment (hennevogel) [inline src/api/lib/tasks/dev.rake:264]: soo why isn't all of this a factory?

File: src/api/lib/tasks/dev.rake (hunk 1 above)
Comment (saraycp) [reply src/api/lib/tasks/dev.rake]: As `response_body` and `response_url` change depending on different statuses, I decided to create factories for workflow runs with different status and refactor the task code a bit.

File: src/api/lib/tasks/dev.rake (hunk 2 above)
Comment (saraycp) [reply src/api/lib/tasks/dev.rake:264]: I added a new commit on top to create the objects to which the artifacts link to.

**Review by dmarcoux — APPROVED** [review]

**Pull request merged**